recipes/anymap/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "anymap" %}
+{% set version = "0.1.0" %}
+{% set python_min = "3.9" %}
```

Comment:
Only set `python_min` if the minimum required version is higher than the conda-forge default. 

```suggestion
```